components/store/CheckoutForm.js

```diff
@@ -0,0 +1,492 @@
+                      return (
+                        <tr key={item.parent}>
+                          <td className="text-right text-space-right">{item.quantity && `${item.quantity}x `}{item.description}{descriptors && ` (${descriptors})`}</td>
+                          <td>{
+                            (item.amount / 100).toLocaleString('en-GB', {
+                              style: 'currency',
+                              currency: order.attributes.currency,
+                              currencyDisplay: 'symbol',
+                            })}
+                          </td>
+                        </tr>
+                      )
+                    })}
+                    <tr>
+                      <td className="text-bold text-right">Total</td>
+                      <td className="text-bold total">
+                        {
+                          (order.attributes.amount / 100).toLocaleString('en-GB', {
+                            style: 'currency',
+                            currency: order.attributes.currency,
+                            currencyDisplay: 'symbol',
+                          })
+                        }
+                      </td>
+                    </tr>
+                  </tbody>
+                </table>
+              </div>
+              <form className="compact center" onSubmit={this._handleOrderConfirm}>
+                {!cardToken && (
+                  <>
+                    <h4>Billing Info</h4>
+                    <ValidatedFormInput
+                      autoComplete="cc-name"
+                      id="name"
+                      label="Cardholder Name"
+                      name="name"
+                      onChange={this._handleCardNameChange}
+                      required
+                      value={cardName} />
+                    <fieldset>
+                      <CardElement onChange={this._handleCardElementChange} />
+                    </fieldset>
+                  </>
+                )}
+                <menu type="toolbar">
+                  <div className="primary">
+                    <button
+                      className="green"
+                      disabled={stageIsValid !== true}
+                      type="submit">
+                      {stageIsValid === true ? 'Submit Order' : stageIsValid}
+                    </button>
+                  </div>
+                  <div className="secondary" />
+                </menu>
+              </form>
+            </>
+          } else if (stage === 3) {
+            <>
+              <h2>Success!</h2>
+              <span>Your Order has been submitted. Be sure to check your email for confirmation!</span>
+              <span>You will receive a shipment confirmation email once your items are shipped!</span>
+              <span>Questions, concerns, or wish to cancel? Contact us at _INSERT_EMAIL_</span>
```

Comment:
I need an email for this here. Secondly do we need to have an additional page written up outlining the return and cancellation policy? If so we can add it to wordpress fairly easily, and I can link to it.

Reply:
Use orders@fuelrats.com  as the email, create a dummy wordpress page to use for your integration. Abs and I will fill it out eventually

Reply:
also pinging @kenneaal for approval of this blurb. Anything you would want to change about it?

Reply:
Your Order has been submitted. Be sure to check your email for confirmation!
Once our quartermaster gets around to shipping your order, you will get a shipment confirmation email. Please note that only larger items sent by parcel have tracking numbers; smaller items like stickers and badges are sent as normal mail, but we'll still let you know when it is posted!
If you have any questions or concerns, please contact us at orders@fuelrats.com. You can find our return and cancellation policy in your order confirmation email.